core/bin/check_dsp

```diff
@@ -62,7 +62,7 @@ foreach my $dir (@DIRS)
   my @file_list = grep {! m/^(\..*|CVS|\.svn)$/} readdir(BASEDIR);
   closedir(BASEDIR);
   my @sd = ();
-  if    ($dir =~ m!\bvcl$!)   { @sd = qw(vcl_sys iso generic gcc egcs gcc-295 gcc-libstdcxx-v3 sgi sunpro win32-vc60 win32-vc70 win32-vc8 emulation borland55 borland56 mwerks stlport internal); }
+  if    ($dir =~ m!\bvcl$!)   { @sd = qw(vcl_sys iso generic gcc egcs gcc-295 gcc-libstdcxx-v3 sgi sunpro win32-vc60 win32-vc70 win32-vc8 emulation stlport internal); }
```

Comment:
Yes.  I already have those patches created.   Just letting them trickle in one at a time.  There is some inter-relations between these compilers, so identifying completely clean patch sets has been difficult.